components/simplifiedGrid/SimplifiedGrid.css

```diff
@@ -0,0 +1,49 @@
+@import url("/components/image/Image.css");
+@import url("/components/card/Card.css");
+
+.grid-col-2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px;
+}
+
+.grid-col-3 {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 10px;
+}
+
+.grid-item {
+  box-shadow: 0 5px 20px -5px rgb(191 192 192);
```

Comment:
A variable can be used here as well.

Reply:
Yes, adding global variable for box shadow, thanks.